README.md

```diff
@@ -1,9 +1,24 @@
 The ev3dev meta repo
 ====================
-This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases),
-the ev3dev [wiki](https://github.com/ev3dev/ev3dev/wiki) and the [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-To learn more about ev3dev, check out our home page at <http://www.ev3dev.org>.
+This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases) and the project-wide  [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-If you are looking for code, check out all of the other repositories at <https://github.com/ev3dev>.
+### Looking for documentation and tutorials? Check out our [website](http://www.ev3dev.org).
 
+Our website hosts docs, guides and tutorials to help you get up-and-running with ev3dev as quickly as possible. Check out the [getting started guide](http://www.ev3dev.org/docs/getting-started/) if you're looking for a place to jump in.
+
+### Searching for code? Have a look at all of the other repositories in the [ev3dev GitHub organization](https://github.com/ev3dev).
+
+This repo just hosts the "meta content" that spans our many codebases and subprojecs. The code itself is managed granularly on a per-project level, and each gets its own repo.
+
+**Central code repositories:**
+- [ev3dev kernel](https://github.com/ev3dev/ev3-kernel): The foundational Linux-based kernel that runs ev3dev.
+- [LEGO device drivers](https://github.com/ev3dev/lego-linux-drivers): Drivers that provide control over LEGO MINDSTORMS devices and related hardware.
+- [Brickman](https://github.com/ev3dev/brickman): Graphical brick manager for ev3dev.
+- [ev3dev website](https://github.com/ev3dev/ev3dev.github.io): The ev3dev website and documentation.
```

Comment:
brickstrap is another core repository.

Also, if you want to build a kernel then ev3dev-build-scripts is what you want. In fact, this is probably the repository that I hand out links for the most, so it might even merit it's own section. "Want to compile a kernel? Go here..."